test/ttmlir/Dialect/TTIR/stablehlo_to_ttir_ExponentialOp.mlir

```diff
@@ -0,0 +1,11 @@
+// REQUIRES: stablehlo
+// RUN: ttmlir-opt --convert-stablehlo-to-ttir %s | FileCheck %s
+#any_device = #tt.operand_constraint<dram|l1|scalar|tile|any_device|any_device_tile>
+module @jit_eltwise_exponential attributes {} {
+  func.func public @test_exponential(%arg0: tensor<13x21x3xf32>) -> tensor<13x21x3xf32> {
+    %0 = stablehlo.exponential %arg0 : tensor<13x21x3xf32>
+    // CHECK: %[[C:.*]] = tensor.empty[[C:.*]]
+    // CHECK: %[[C:.*]] = "ttir.exponential"[[C:.*]]
+    return %0 : tensor<13x21x3xf32>
+  }
+}
```

Comment:
For new OPs you also need to add a test for TTIR->TTNN conversion in `test/ttmlir/Dialect/TTNN/` folder. Sorry I missed this yesterday.

Reply:
Ok thanks, added ExpOp test for TTNN.